.github/workflows/refapp-2x-clinical-visit.yml

```diff
@@ -4,9 +4,7 @@ on:
     branches: [master]
   pull_request:
     branches: [master]
-    paths:
-      - "qaframework-bdd-tests/src/**/*.*"
-      - "qaframework-bdd-tests/package.json"
+    
```

Comment:
I think we can get rid of this line here

Reply:
Resolved

Reply:
> Resolved

Is there any commit made to resolve the review above?

Reply:
Opps sorry my internet had not pushed this.